packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -63,14 +69,24 @@ function NotificationItem({
     >
       {onClick && (
         <Link href={targetUrl} passHref>
-          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/anchor-has-content, jsx-a11y/no-static-element-interactions */}
           <a
+            role="link"
             aria-label="Open notification"
             className="absolute inset-0 z-0"
             onClick={onClick}
             title="Open notification"
             data-testid="openNotification"
-          />
+            tabIndex={0}
+            onKeyDown={(e) => {
+              if (e.code === 'Space') {
```

Comment:
should we add this to the elements enum? https://github.com/dailydotdev/apps/blob/7c546fbf1006a430b9c54586ffd1094a30a2d486/packages/shared/src/lib/element.ts#L3-L5

Reply:
Yeah sure, done :D